### PR TITLE
various updates

### PR DESCRIPTION
**Pull request by JensTrier**
Updating the following:

- New output fields from Get-CsPhoneNumberAssignment
- New example and note in Get-CsAutoAttendantHolidays
- Noted that cmdlet has been removed for Test-CsOnlineLisCivicAddress
- Updating *-CsCallingLineIdentity and *-CsOnlineLisCivicAddress with links etc
- Removing deleted parameters from New/Set-CsTeamsCallingPolicy

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md [hunk 1]
```diff
@@ -5,85 +5,72 @@ applicable: Microsoft Teams, Skype for Business Online
...
+Grant-CsCallingLineIdentity [[-PolicyName] <string>] [-Global] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+### Identity
+```
+Grant-CsCallingLineIdentity [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-You can either change or block the Caller ID (also called a Calling Line ID) for a user. By default, the user's phone number can be seen when that user makes a call to a PSTN phone, or when a call comes in. You can create a Caller ID policy to provide an alternate displayed number, or to block any number from being displayed; you can then apply the Caller ID policy to a specific user.
+You can either a Caller ID policy to a specific user or you can set the Global policy instance.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:30]:
```suggestion
You can either create a Caller ID policy for a specific user or you can set the Global policy instance.
```

Comment (JohanFreelancer9) [reply skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:30]: @JensTrier, I am not sure about this. Please feel free to change it. However, the original line is incomplete.

File: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md [hunk 2]
```diff
@@ -93,53 +80,46 @@ Accept pipeline input: False
...
+Position: 1
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -PassThru
-Returns the results of the command. By default, this cmdlet does not generate any output.
+### -Identity
+The Identity of the user to whom the policy is being assigned. User Identities can be specified using the user's SIP address, the user's user principal name (UPN) or,
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:100]:
```suggestion
The Identity of the user to whom the policy is being assigned. User Identities can be specified using the user's SIP address, the user's user principal name (UPN), or
```

File: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md [hunk 3]
```diff
@@ -149,13 +129,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -WhatIf
-The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes. 
+### -Confirm
+The Confirm switch causes the command to pause processing, and requires confirmation to proceed.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsCallingLineIdentity.md:133]:
```suggestion
The Confirm switch causes the command to pause processing and requires confirmation to proceed.
```

File: skype/skype-ps/skype/New-CsCallingLineIdentity.md [hunk 4]
```diff
@@ -103,15 +101,13 @@ Accept wildcard characters: False
 ```
 
 ### -CallingIDSubstitute
-PARAMVALUE: Anonymous | Service | LineUri | Resource
-
-The CallingIDSubstitute parameter lets you specify an alternate Caller ID. The default value is LineUri.
+The CallingIDSubstitute parameter lets you specify an alternate Caller ID. The default value is LineUri. Supported values are Anonymous, Service, LineUri and Resource.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallingLineIdentity.md:104]:
```suggestion
The CallingIDSubstitute parameter lets you specify an alternate Caller ID. The default value is LineUri. Supported values are Anonymous, Service, LineUri, and Resource.
```

File: skype/skype-ps/skype/New-CsCallingLineIdentity.md [hunk 5]
```diff
@@ -264,14 +214,14 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -WhatIf
-The WhatIf switch causes the command to simulate its results. By using this switch, you can view what changes would occur without having to commit those changes. 
+### -Confirm
+The Confirm switch causes the command to pause processing, and requires confirmation to proceed.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallingLineIdentity.md:218]:
```suggestion
The Confirm switch causes the command to pause processing and requires confirmation to proceed. 
```

File: skype/skype-ps/skype/New-CsOnlineLisCivicAddress.md [hunk 6]
```diff
@@ -330,14 +310,15 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -StreetName
-Specifies the street name of the new civic address.
+### -PreDirectional
+Specifies the directional attribute of the new civic address which precedes the street name.
+For example, "425 NE Smith Avenue ".
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsOnlineLisCivicAddress.md:315]:
```suggestion
For example, "425 NE Smith Avenue".
```

**Review by dariomws — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 7]
```diff
@@ -367,17 +371,21 @@ The ID of the object the phone number is assigned to.
...
+
 ### CivicAddressId
 The ID of the CivicAddress assigned to the phone number.
 
-
 ### IsoCountryCode
 The ISO country code assigned to the phone number.
 
+### IsoSubDivision
+The sub division within the country assigned to the phone number, for example the state for US phone numbers.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:384]:
```suggestion
The subdivision within the country assigned to the phone number, for example, the state for US phone numbers.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the few minor suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the few minor suggested changes. Many thanks.

Thanks @JohanFreelancer9 & Done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**